src/power.rs

```diff
@@ -81,3 +97,14 @@ pub fn read_power_source() -> Result<bool, Error> {
 
     return Ok(pwr_str == "1");
 }
+
+pub fn get_best_path(paths: Vec::<&str>) -> Result<&str, Error> {
+
+    for path in paths.iter() {
+        if Path::new(path).exists() {
+            return Ok(path);
+        }
+    }
+
+    return Err(Error::Unknown);
```

Comment:
Maybe add a thing to let the user know that their system is unsupported and tell them to submit an issue in the repo

Reply:
Great idea!

Reply:
Should we keep the little script in the github gist or should we move it this repo? I'm good with either